package exceptions_article_second_part;

public class App5 {
    public static void main(String[] args) {
        //f(); // тут ошибка компиляции
    }

    public static void f() throws Exception {
    }
}
